tests/standalone_tests/test_parser.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+import tube.settings as config
+
+from tube.utils.dd import init_dictionary
+from unittest.mock import patch
+from tests.util import mock_dictionary_url, initialize_mappings
+from tube.etl.indexers.injection.parser import Parser as InjectionParser
```

Comment:
[Per PEP, by convention](https://peps.python.org/pep-0008/#imports):
> Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
>
> You should put a blank line between each group of imports.
```suggestion
import pytest

from unittest.mock import patch

from tests.util import mock_dictionary_url, initialize_mappings
from tube.etl.indexers.injection.parser import Parser as InjectionParser
import tube.settings as config
from tube.utils.dd import init_dictionary
```